lessons/Data-Science/measures-of-center/langs/en-us/index.adoc

```diff
@@ -276,66 +191,54 @@ Consider the following three datasets:
 - The mode of the second dataset is 2, since 2 appears more than any other number.
 - The modes (plural!) of the last dataset are 1 and 4, because 1 and 4 both appear more often than any other element, and because they appear equally often.
 
+
 @slidebreak
 
-@opt{The Modes section of @opt-printable-exercise{mean-median-modes.adoc} includes a printed version of the upcoming list.}
+Can you find the mode(s) of this dataset?
 
-@lesson-instruction{
-Take a minute to identify the mode(s) for each of the following datasets:
+```
+red, green, red, yellow, blue, red, purple, purple
+```
 
-- 11, 3, 7, 4, 5  
-- 5, 7, 11, 11, 7, 7    
-- 2, 3, 5, 4, 3, 7, 4
+@teacher{The mode here is red, which appears three times on the list. Highlight for students that *yes*, we can find the mode of a categorical dataset!}
 
-}
 
 @slidebreak
 
-@ifproglang{pyret}{
-Pyret has a function that will compute the modes of any quantitative column in a Table. 
+@lesson-instruction{
+- Complete the Modes section of @opt-printable-exercise{mean-median-modes.adoc}.
+- Compare your answers with a partner's. Correct any mistakes.
+}
 
-@hspace{2em} @show{ (contract "modes" '("Table" "String") "List" )}
 
-_Note: `List` is a new data type!_
 
-Let's test it out!
-}
+=== Synthesize
 
-@ifproglang{codap}{
-The easiest way to determine modes in CODAP is to sort a column. Do this by clicking on the column name and then selecting from the drop-down menu either Sort option. Scan the column to see which values are the most common.
-}
+@QandA{
 
-=== Investigate
-@lesson-instruction{
-- Turn to @printable-exercise{summarizing-columns-moc.adoc} @ifproglang{pyret}{and use the code provided} to compute and record the `modes` of the `pounds` column.
-- Then complete the remaining questions in the *Summarizing the `Pounds` Column* section.
-}
+@Q{If you heard that the mean age of students in a kindergarten class was 21, would you be surprised? Why or why not?}
```

Comment:
Curious about the choice of 21 here. Why not something big, but _somewhat_ plausible, like 8 or 9. That leaves the door open for a kid to hypothesize that there's an 18 year old in the class, demonstrating a deeper understanding of mean?

Reply:
re: curious about the choice of 21... that is in the fall2024 version of the lesson! So, that's where it came from. 

Anyhow. I will happily change it to 8 or 9.

Reply:
(Although I wonder if 21 was trying to mislead students because it is a reasonable number of students per class.)